content/cloud-computing/concepts/iaas/iaas.md

```diff
@@ -0,0 +1,20 @@
+---
+Title: 'IaaS'
+Description: 'Infrastructure as a service (IaaS) is a cloud computing model that offers remote computer, storage, and networking resources that are available on demand.'
+Subjects:
+  - 'Computer Science'
+  - 'Networking'
+Tags:
+  - 'Networking'
+  - 'Cloud'
+  - 'Infrastructure'
+CatalogContent:
+  - 'foundations-of-cloud-computing'
+  - 'paths/back-end-engineer-career-path'
+---
+
+**Infrastructure as a service (IaaS)** is a cloud computing model that offers remote computer, storage, and networking resources that are available on demand. These services are typically offered on a pay-as-you-go basis. The user is given control of virtual machines that are guests on a cloud server running a hypervisor such as Oracle VirtualBox or VMware ESXi. They are allowed to run arbitrary software on these virtual machines, including operating systems.
+
+IaaS is also provided with an [API](https://www.codecademy.com/resources/docs/general/api) to control the low-level details of the underlying network infrastructure: things such as backups, partitioning, security, physical resources, and so on. While the user does not manage or control the underlying cloud infrastructure, they have granular control over operating systems, storage, deployed applications, as well as some networking components such as host firewalls.
```

Comment:
IaaS provides an API maybe instead of is also provided by. Also this first sentence is run on. 

IaaS provides an [API](https://www.codecademy.com/resources/docs/general/api) to control the low-level details of the underlying infrastructure. While they do not have direct access to the infrastructure, users can configure:

Reply:
Followed by a list of the things you list in the paragraph.

Reply:
```suggestion
IaaS provides an [API](https://www.codecademy.com/resources/docs/general/api) to control the low-level details of the underlying network infrastructure (e.g., backups, partitioning, security, and physical resources). While they do not have direct access to the infrastructure, users can configure:

- Operating systems
- Storage
- Deployed applications
- Networking components (e.g., host firewalls)
```